pkg/config/setup/config.go

```diff
@@ -756,6 +758,8 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("admission_controller.inject_config.dogstatsd_socket", "unix:///var/run/datadog/dsd.socket")
 	config.BindEnvAndSetDefault("admission_controller.inject_config.type_socket_volumes", false)
 	config.BindEnvAndSetDefault("admission_controller.inject_tags.enabled", true)
+	config.BindEnvAndSetDefault("admission_controller.inject_tags.enabled_namespaces", []string{})
+	config.BindEnvAndSetDefault("admission_controller.inject_tags.disabled_namespaces", []string{})
```

Comment:
Should we add those 4 new settings to the configuration template (`pkg/config/config_template.yaml`) ?

Reply:
We're still debating if we need to actually expose these options or not. I added this comment as a follow up to make sure it's not dropped: https://datadoghq.atlassian.net/browse/INPLAT-472